korebuild-lock.txt

```diff
@@ -1,2 +1,2 @@
-version:2.1.0-preview2-15683
-commithash:54af97ac56dfdc495c2514c6916fe4ca79910939
+version:2.1.0-preview2-15684
```

Comment:
I assume this has an upgraded version of the Feed package?

Reply:
Yup